loki-dev-client.js

```diff
@@ -1 +1 @@
-console.log("hello")
\ No newline at end of file
+console.log("hello world")
```

Comment:
This change would break backwards compatibility with previous versions. Can we make this less verbose?